.github/workflows/build-on-main.yml

```diff
@@ -0,0 +1,100 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+##### README #####
+#
+# This CI action runs the release crates as downloaded from crates.io. It does not block PRs
+# and it is okay if it is flaky from time to time.
+
+name: Build Daily on Main
+
+on:
+  push:
+    branches: [ main ]
+  schedule:
+    # Run this CI every day at 13 UTC / 5AM PST / 2PM CET
+    - cron:  '0 13 * * *'
```

Comment:
@Manishearth Thoughts on adding a new file to collect tasks that we only want to run once a day?

Reply:
This is at 13 and the other is at 14

Reply:
+1 go for it yolo

Reply:
You're running this on push and at 13 UTC?

Reply:
I moved it to 12 UTC and removed the running on main merge